src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +136,24 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+    //this method is used to get the serverinfo object from the server
```

Comment:
This comment isn't very useful - I can infer what it tells me from the method signature. Consider providing an overview of the method - how it does what it does, what possible exceptions it could encounter, etc.

If you want to go the extra mile, use [Javadoc](https://en.wikipedia.org/wiki/Javadoc).